pkg/logs/internal/framer/framer.go

```diff
@@ -164,6 +164,7 @@ func (fr *Framer) Process(input *message.Message) {
 			// chop off contentLenLimit raw bytes and output them
 			if len(buf) >= contentLenLimit {
 				content, rawDataLen = buf[:contentLenLimit], contentLenLimit
+				input.ParsingExtra.IsTruncated = true
```

Comment:
I believe this is correct, though it does have me wondering if we should also be adding the `...TRUNCATED...` text that the tailers add when they truncate. To me, this seems like it would be more consistent.

Also this gives even more credence to my idea in another comment that truncation should be top-level field rather than part of `ParsingExtra`.